RELEASING.md

```diff
@@ -40,3 +40,33 @@ You should have no local changes and tests should be passing.
 1. Close the milestone for this release. https://github.com/tigera/operator/milestones
 
 1. Go to https://github.com/tigera/operator/milestones and create any new milestones that should exist (e.g., next patch release)
+
+## Creating a release on RH Catalog
+
+1. After the semaphore job in the releasing steps is complete, and images have been tagged and pushed, checkout the tag you released.
+
+1. Create the ClusterServiceVersion (CSV), using the tag version for VERSION and the version that this replaces in PREV_VERSION.
```

Comment:
Should we include any notes about needing updates if any new Calico CRDs were added since previous releases?

Reply:
Yes, thanks: 556c0655aca9c858cdac91e71b563de44a1ff432